locales-pending/premium.ftl

```diff
@@ -298,6 +298,29 @@ data-broker-profiles-estimated-time = Est. time to complete: { $estimated_time }
 # $exposure_reduction is the percentage of exposures that are data brokers.
 data-broker-profiles-exposure-reduction = Exposure reduction: { $exposure_reduction }%
 
+# Welcome to Premium Data Broker Profiles
+
+# Exposure reduction chart
+# The number inside <nr> will be displayed in a large font,
+# the label inside <label> will be shown in a smaller font.
+# Variables:
+#   $nr (number) - % of exposures reduced for the user
+exposure-reduction-chart-heading = <nr>{ $nr }</nr> <label>% exposures will be reduced</label>
```

Comment:
Q: what does this look like?

I see the following code below:

```js
l10n.getString("exposure-reduction-chart-heading", {
                nr: props.exposureReduction,
              })
              .replace("<nr>", "")
              .replace("</nr>", "")
              .replace("<label>", "")
              .replace("</label>", "")}
```

So would this end up returning a localized string like this?
> "6 % exposures will be reduced"

Oh! The comments say that the segments will be in different fonts, so maybe the space after the number and before the "%" doesn't matter as much.

Reply:
outdated now